demo/src/screens/interactableComponentScreens/DrawerScreen.js

```diff
@@ -50,15 +61,45 @@ export default class DrawerScreen extends Component {
     }
   }
 
+  // Measure item text to calculate items' widths
+  async setRightItemsWidths() {
```

Comment:
just for code cleanness, can we unified `setRightItemsWidths` & `setLeftItemWidth` to one method that knows to handle either one item or list of items. 
it just the code looks a bit repetitive

Reply:
Done